src/org/camunda/latera/bss/connectors/HOMS.groovy

```diff
@@ -169,16 +169,17 @@ class HOMS {
     return attachFiles([file], prefix, save)
   }
 
-  void sendTaskEvent(String taskId, String eventName) {
+  void sendTaskEvent(String taskId, String eventName, List<String> users) {
     LinkedHashMap body = [
-      event_name: eventName
+      event_name: eventName,
+      users:      users
     ]
 
     logger.info("/ Sending event ${eventName} for task ${taskId} ...")
 
     this.http.sendRequest(
       'put',
-      path: "events/tasks/${taskId}",
+      path: "/widget/events/tasks/${taskId}",
```

Comment:
These events are about widget not HOMS in fact so I think they must be routed to appropriate controller.